cmd/pbstatuser/main.go

```diff
@@ -151,16 +151,24 @@ func checkSourceAvailabilityAWS(ctx context.Context) {
 				Identity:     s.Identity,
 				ResourceType: "Application",
 			}
-			_, err = clients.GetEC2Client(ctx, &s.Authentication, "")
+			ec2Client, err := clients.GetEC2Client(ctx, &s.Authentication, "")
 			if err != nil {
 				sr.Status = kafka.StatusUnavailable
 				sr.Err = err
-				logger.Warn().Err(err).Msg("Could not get aws assumed client")
-				chSend <- sr
+				logger.Warn().Err(err).Msgf("Could not get aws assumed client %s", err)
```

Comment:
Please do not send error messages to messages because of Sentry.

https://github.com/RHEnVision/provisioning-backend/pull/488